packages/manager/apps/catalog/e2e/step-definitions/filter-and-search.step.ts

```diff
@@ -0,0 +1,80 @@
+import { expect } from '@playwright/test';
+import { Given, When, Then } from '@cucumber/cucumber';
+import { ICustomWorld } from '../../../../../../playwright-helpers';
```

Comment:
Same as previous

Reply:
No it doesn't work
The alias in tsconfig.json is not taken into account